src/library_fs.js

```diff
@@ -413,6 +413,7 @@ mergeInto(LibraryManager.library, {
         FS.FSStream.prototype = {};
         // compatibility
         Object.defineProperties(FS.FSStream.prototype, {
+        FS.FSStream = /** @constructor */ function(){};
```

Comment:
this doesn't look right - maybe it was pasted in the wrong place?

Reply:
Ops, not sure what I was doing there!